src/java/picard/sam/markduplicates/EstimateLibraryComplexity.java

```diff
@@ -82,20 +82,31 @@
-                "they match each other with no gaps and an overall mismatch rate less than or equal to " +
-                "MAX_DIFF_RATE (0.03 by default).\n\n" +
-                "Reads of poor quality are filtered out so as to provide a more accurate estimate. The filtering " +
-                "removes reads with any no-calls in the first N bases or with a mean base quality lower than " +
-                "MIN_MEAN_QUALITY across either the first or second read.\n\n" +
-                "Unpaired reads are ignored in this computation.\n\n" +
-                "The algorithm attempts to detect optical duplicates separately from PCR duplicates and excludes " +
-                "these in the calculation of library size. Also, since there is no alignment to screen out technical " +
-                "reads one further filter is applied on the data.  After examining all reads a Histogram is built of " +
-                "[#reads in duplicate set -> #of duplicate sets] all bins that contain exactly one duplicate set are " +
-                "then removed from the Histogram as outliers before library size is estimated.",
+        usage = "Attempts to estimate library complexity from sequence of read pairs alone (unmapped reads). Does so by" +
+                " sorting all reads by the first N bases (5 by default) of each read and then comparing reads with the" +
+                " first N bases identical to each other for duplicates.  Reads are considered to be duplicates if they" +
+                " match each other with no gaps and an overall mismatch rate less than or equal to MAX_DIFF_RATE " +
+                "(0.03 by default).  <br /><br />" +
+                "" +
+                "Reads of poor quality are filtered out so as to provide a more accurate estimate. " +
+                "The filtering removes reads with any no-calls in the first N bases or with a mean base quality" +
+                " lower than MIN_MEAN_QUALITY across either the first or second read.  Unpaired reads are ignored in" +
+                " this computation.  <br /><br /> " +
+                "" +
+                "The algorithm attempts to detect optical duplicates separately from PCR duplicates and excludes" +
+                " these in the calculation of library size.  Also, since there is no alignment to screen out technical" +
+                " reads one further filter is applied on the data.  After examining all reads a Histogram is built" +
+                " of [#reads in duplicate set -> #of duplicate sets] all bins that contain exactly one duplicate set" +
+                " are then removed from the Histogram as outliers before library size is estimated. <br />" +
+                "" +
+                "<h4>Usage example:</h4>" +
+                "<pre>" +
+                "java -jar picard.jar EstimateLibraryComplexity \\<br />" +
+                "     -I=Mybamfile.bam \\<br />" +
+                "     -O=Estlibmetrics.txt" +
```

Comment:
General comment -- I think the naming logic would be more clear (especially to non-native speakers) if you used underscores to separate the word components, e.g. "Estlibmetrics" -> "est_lib_metrics"